### DISCUSSION TITLE
check

### DISCUSSION THREAD

**Discussion by rocharhito** [community/community#126627] [Other Feature Feedback, Questions, & Ideas]
[Out with the old check-in, and in with the NEW](https://github.com/community/community/discussions/114621)

**Comment by ebndev** [comment]
Hi @rocharhito, thanks for being a part of the GitHub Community!

We appreciate you wanting to highlight one of our Community Check-Ins however, I will be closing this discussion as it is considered a duplicate. If you have any thoughts you'd like to share, please feel free to comment in the original post :)
